### ISSUE TITLE
Unabridge camping-unabridged even more.

### ISSUE THREAD

**Issue by karloscarweber** [camping/camping#113]
I think it would be super great to Add more documentation and explanation to `camping-unabridged.rb`. Right now I'm trying to diagnose a bug where multiple apps won't load and it's a bit difficult to follow the execution chain and where everything goes.

I suggest breaking up certain methods and giving variables more verbose names in _unabridged_, while maintaining the strict syntax of camping condensed.

A flow chart of what happens when we call camping, and in what steps, and why, would also make it easier to understand the code, and why it's doing what it's doing.

So, to recap, I want to:

- Write more verbose names
- Break up some methods.
- Create a flow chart of actions that Camping takes.

I believe that doing this will make maintenance and development easier. It will also be instructional to newcomers reading the source code. I mean I'm a newcomer and I've learned so much just by trying to piece it together.

**Comment by karloscarweber** [comment]
One problem I've been running into while debugging is trying to understand which routes or controllers are mounted. I want to write a small utility that spits out the routes for a mount. Like:
```bash
=> Camping routes nuts.rb
=> Index, I, Auth, Whatever 
```

While in irb this code has been useful to inspect at least the classes in the controllers. 
```ruby
Auth::Controllers.constants.each do |x| 
    puts "#{x}"
end
```

I'll keep iterating on this.

**Comment by karloscarweber** [comment]
I think that making a small module that gives us useful Camping inspection tools is a really great idea. That's what I'm going to work on to bring some of these features to Camping.
